lib/datapacksjob.js

```diff
@@ -3291,13 +3291,14 @@ DataPacksJob.prototype.datapackOpen = async function (jobInfo) {
         for (const obj of jobInfo.queries) {
             if (obj.VlocityDataPackType === jobInfo.type) {
                 let query = obj.query.replace(/%vlocity_namespace%__/g, this.vlocity.namespacePrefix);
-                query += query.includes('WHERE') ? ` AND Name = '${jobInfo.name}'` : ` Where Name = '${jobInfo.name}'`;
+                let additionalCondition = `${this.vlocity.namespace}__Type__c = '${jobInfo.omniType}' AND ${this.vlocity.namespace}__SubType__c = '${jobInfo.omniSubType}'`
```

Comment:
@ManasAgarwalVlocity -- Does htis conflict with your changes? I think the Type / Subtype would only be for OmniScript / IP, so this should be separated by type.

Reply:
Yes. If DataPackOpen is still a cli command, not specific to this feature, then it will conflict as it is to open designer by datapack name. Will also require command description update in vlocitycli.js to set input expectations from user.  Also for OmniScript, language may also be required to get unique record.